pkgs/os-specific/linux/nixos-rebuild/default.nix

```diff
@@ -19,7 +21,7 @@ substituteAll {
   nix_x86_64_linux = fallback.x86_64-linux;
   nix_i686_linux = fallback.i686-linux;
   nix_aarch64_linux = fallback.aarch64-linux;
-  path = lib.makeBinPath [ coreutils gnused gnugrep ];
+  path = lib.makeBinPath [ coreutils gnused gnugrep jq util-linux ];
```

Comment:
Interestingly jq is already in $PATH in my nixos-rebuild. Where does it come from when it's not in this list?

Reply:
Interesting... It previously was in this list, but it got removed with c75bc3abc760a07eb7afc0be26eb1813a9867a84 because it was no longer in use.